tests/software_tests/can/packet/test_consecutive_frame.py

```
@@ -0,0 +1,590 @@
+
+    def teardown_method(self):
+        self._patcher_validate_nibble.stop()
+        self._patcher_validate_raw_byte.stop()
+        self._patcher_validate_raw_bytes.stop()
+        self._patcher_can_dlc_handler.stop()
+        self._patcher_can_addressing_information.stop()
+
+    # is_consecutive_frame
+
+    @pytest.mark.parametrize("addressing_format, raw_frame_data, ai_data_bytes_number, expected_output", [
+        (Mock(), [CONSECUTIVE_FRAME_N_PCI << 4] + list(range(100, 163)), 0, True),
```

Ruff (0.12.2)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)